VCFoodDeliveryApp/Models/Cart.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace VCFoodDeliveryApp.Models
+{
+    public class Cart
+    {
+        [StringLength(450, ErrorMessage = "Too many users created")]
+        public string Id { get; set; }
+
+
+        public string ApplicationUserId { get; set; }
+
+
+        public DateTime DeliveryTime { get; set; }
```

Comment:
Expected usage is unclear, but it seems like you might want this to be null until the user has told you when they want delivery.

Reply:
After taking some time to think about this: I think we will need multiple time properties. We will need to add a food prep/cook time and a distance to travel time or something similar. I think based on how long food takes to be done + how long it takes to drive the distance to the delivery location we could provide some buttons/choices with times. 

Possible delivery times:
7:15
7:30
7:45

Reply:
Are these things _cart_ properties or _order_ properties that you would set while you're checking out.